github/repos_collaborators.go

```diff
@@ -10,10 +10,25 @@ import (
 	"fmt"
 )
 
+// ListCollaboratorsOptions specifies the optional parameters to the
+// RepositoriesService.ListCollaborators method.
+type ListCollaboratorsOptions struct {
+	// Affiliation specifies how collaborators should be filtered by their affiliation.
+	// Possible values are:
+	//     outside - All outside collaborators of an organization-owned repository
+	//     direct - All collaborators with permissions to an organization-owned repository, regardless of organization membership status
```

Comment:
It might be nice to wrap this line, like so:

```Go
// Possible values are:
//     outside - All outside collaborators of an organization-owned repository
//     direct - All collaborators with permissions to an organization-owned repository,
//              regardless of organization membership status
//     all - All collaborators the authenticated user can see
```

Precedent:

https://github.com/google/go-github/blob/5f3187e17912ee2e3fbe2fa5bdada9d577d25e83/github/orgs_members.go#L26-L30